data/blog/intro-to-entity-framework-core-in-net-6.md

```diff
@@ -0,0 +1,586 @@
+---
+title: Intro to Entity Framework Core in .NET 6
+date: '2023-07-19'
+tags: ['C#', '.NET', 'efcore', 'Csharp']
+draft: false
+summary: Entity framework is a .NET tool to interact with various kind of databases, providing the same api for each of these database providers. In this blog post, you will see how to set un Entity Framework and build your first CRUD operation over a Sqlite database
```

Comment:
```suggestion
Entity framework is a .NET tool to interact with various kind of databases, providing the same API for each of these database providers. In this blog post, you will see how to set an Entity Framework and build your first CRUD operation over a SQLite database
```

There are other parts of the article where some of these typos/bad capitalization happen. Can you update them as well?